main.c

```diff
@@ -286,6 +295,15 @@ static int parent(int sock, int ready_fd, int exit_fd, const char *api_socket,
         }
     }
 #endif
+    if (cfg->vhwaddress_len > 0) {
+        char mac_address[64];
+        memset(mac_address, 0, sizeof(mac_address));
+        if (hwaddr_data_to_hexstring((unsigned char *)cfg->vhwaddress.sa_data,
+                                     cfg->vhwaddress_len, mac_address,
+                                     sizeof(mac_address)) > 0) {
+            printf("* MAC address:     %s\n", mac_address);
```

Comment:
I don't think we need to have `hwaddr_data_to_hexstring`.

Could be like this:
```c
uint8_t *mac=cfg->vhwaddress.sa_data;
printf(" * MAC address:    %02X:%02X:%02X:%02X:%02X:%02X\n",
  mac[0], mac[1], mac[2], mac[3], mac[4], mac[5]);
```

Reply:
This has been done.  Resolving conversation.